EventListener/Login.php

```diff
@@ -63,6 +67,21 @@ public function setCustomerId($value)
         $this->options['customerId'] = $value;
     }
 
+    protected function getUserIdentifier(TokenInterface $token)
+    {
+        $tokenUser = $token->getUser();
+
+        if ($tokenUser instanceof eZUser || !$tokenUser instanceof UserInterface) {
+            return $tokenUser->getAPIUser()->id;
+        }
+
+        if (method_exists($tokenUser, 'getId')) {
+            return $tokenUser->getId();
+        }
```

Comment:
For me as we tried to do before
```php
$tokenUser->getAPIUser()->id;
```
I wanted to give the ability to send an ID if possible.

I would keep it like that OR I would remove the $tokenUser->getAPIUser()->id and I would return all the time the getUsername()

What do you think ?